protocol/signalfx/spanobfuscation/tagmatchrule.go

```diff
@@ -56,7 +56,8 @@ func getRules(ruleConfigs []*TagMatchRuleConfig) ([]*rule, error) {
 	return rules, nil
 }
 
-func getGlob(pattern string) glob.Glob {
+// GetGlob takes a config and returns a gobwas Glob
```

Comment:
You may want to mention here that it returns a Glob with all special characters escaped besides "*"

Reply:
@sstewart-signalfx I've updated the pr with your recommendation